### DISCUSSION TITLE
Idea: Delay

### DISCUSSION THREAD

**Discussion by valki2** [beeyev/Mikrotik-RouterOS-automatic-backup-and-update#6] [Ideas]
Nice script - pretty much exactly what I was looking for... thank you! 👍 

One small addition:  In order to be able to handle bulk upgrades of several devices, it would be nice to have a delay between downloading  the upgrades and rebooting. That way no device is cut off the internet while downloading and the overall downtime can be minimized due to parallelization of the reboots :)

**Comment by beeyev** [comment]
This is a good idea, thank you. I will implement that.
